docs/mkcloud.md

```diff
@@ -166,13 +183,17 @@ you can run the "prepare" step in mkcloud using the `cache_clouddata=1`
 environment variable set. This will create a cache under `$cache_dir` (set to
 `/var/cache/mkcloud` by default) of images and repositories needed during
 deployment. Make sure your `$cache_dir` partition has enough free space to
-store all the repos. You should always monitor the space on that partition and
-adjust accordingly.
+store all the repos. At this time, the cache will require around 23G. You
+should always monitor the space on that partition and adjust accordingly.
 
 Note that you will need to run the "prepare" step again (with VPN access) if
 your `$cloudsource` environment variable had changed. For example, if you are
-developing on "develcloud8" and "develcloud7" in parallel, you will need to run the
-"prepare" step for each separately.
+developing on "develcloud8" and "develcloud7" in parallel, you will need to run
+the "prepare" step for each separately.
```

Comment:
Why? This sounds like a bug or a misunderstanding.
$cloudsource should not change.

> you will need to run "prepare" step for each separately.

Each step should be run in the same configuration context - controlling several clouds from the same shell is error-prone unless you use wrappers that hold and source your config (which is recommended anyway).